### PR TITLE
Update set-up-atp-safe-links-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
added more information on safelinks

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ description: "Set up Safe Links policies to protect your organization from malic
 > This article is intended for business customers who have [Office 365 Advanced Threat Protection](office-365-atp.md). If you are a home user looking for information about Safe Links in Outlook, see [Advanced Outlook.com security](https://support.office.com/article/882d2243-eab9-4545-a58a-b36fee4a46e2).
 
 [ATP Safe Links](atp-safe-links.md), a feature of [Office 365 Advanced Threat Protection](office-365-atp.md) (ATP), can help protect your organization from malicious links used in phishing and other attacks. If you have the necessary [permissions for the Office 365 Security &amp; Compliance Center](permissions-in-the-security-and-compliance-center.md), you can set up ATP Safe Links policies to help ensure that when people click web addresses (URLs), your organization is protected. Your ATP Safe Links policies can be configured to scan URLs in email and URLs in Office documents.
+
+With ATP Safe Links enabled, if a user clicks on a link in an email and the URL has been blocked by your organization’s custom blocked URL list or if the URL is determined to be malicious, then a warning page opens.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-atp-safe-links-policies.md]:
```suggestion
With ATP Safe Links enabled, if a user clicks on a link in an email and the URL has been blocked by your organization’s custom blocked URL list or if the URL is determined to be malicious, a warning page will open.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1289.

Thank you.

cc: @kenwith

**Pull request merged**